### ISSUE TITLE
Change position

### ISSUE THREAD

**Issue by PG-Andrey** [pureqml/qmlcore#56]
In the starting example "Hello world!", I decided to make the text clickable.
````    
Text {
  id: testText;
  anchors.fill: context;

  text: 'Hello, world!';
  font.pixelSize: 32;
  verticalAlignment: Text.AlignVCenter;
  horizontalAlignment: Text.AlignHCenter;

  MouseArea {
    anchors.fill: parent;

    onClicked: {
        testText.text = 'Test text';
    }
  }
}
````
As a result, the inscription itself appears everywhere, but only not in the center! But the method itself for changing the text works out.

**Comment by comrat** [comment]
Good evening!
Yes it's bug.
It will be fixed soon

**Comment by whoozle** [comment]
Hello

As a temporary workaround I can suggest you to rewrite your code as follows:

```
Item {
  id: testText;
  anchors.fill: context;

  Text {
    anchors.fill: parent;
    text: 'Hello, world!';
    font.pixelSize: 32;
    verticalAlignment: Text.AlignVCenter;
    horizontalAlignment: Text.AlignHCenter;
  }

  MouseArea {
    anchors.fill: parent;

    onClicked: {
        testText.text = 'Test text';
    }
  }
}
```

The problem is that any child of Text div affects its scroll size, we will try to circumvent this.

**Comment by whoozle** [comment]
Also I can suggest you using alternative approach for handling clicks:
```
Text {
  id: testText;
  anchors.fill: context;

  text: 'Hello, world!';
  font.pixelSize: 32;
  verticalAlignment: Text.AlignVCenter;
  horizontalAlignment: Text.AlignHCenter;

  ClickMixin { }
    
  onClicked: {
      testText.text = 'Test text';
  }
}
```
It is easier and more readable.

**Comment by whoozle** [comment]
fixed ac811f6063b26b3540d672a90775614f58b37691
